pom.xml

```diff
@@ -32,6 +32,33 @@
   </dependencies>
   <description>A location service for Java, identify a city, state or country by multiple terms.
   </description>
+  <modules>
+    <module>buildtools</module>
+    <module>location4j</module>
+  </modules>
+
+  <properties>
+    <maven.compiler.source>21</maven.compiler.source>
+    <maven.compiler.target>21</maven.compiler.target>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <junit-jupiter.version>5.10.3</junit-jupiter.version>
+    <assertj-core.version>3.26.3</assertj-core.version>
+    <maven-surefire-plugin.version>3.3.1</maven-surefire-plugin.version>
+    <jacoco-maven-plugin.version>0.8.12</jacoco-maven-plugin.version>
+    <sonar.organization>tomaytotomato</sonar.organization>
+    <sonar.host.url>https://sonarcloud.io</sonar.host.url>
+    <sonar-maven-plugin.version>4.0.0.4121</sonar-maven-plugin.version>
+    <sonar.java.coveragePlugin>jacoco</sonar.java.coveragePlugin>
+    <sonar.dynamicAnalysis>reuseReports</sonar.dynamicAnalysis>
+    <sonar.coverage.jacoco.xmlReportPaths>${project.build.directory}/site/jacoco/jacoco.xml
+    </sonar.coverage.jacoco.xmlReportPaths>
+    <sonar.language>java</sonar.language>
+    <central-publishing-maven-plugin.version>0.5.0</central-publishing-maven-plugin.version>
+    <maven-gpg-plugin.version>3.2.4</maven-gpg-plugin.version>
+    <maven-deploy-plugin.version>3.1.2</maven-deploy-plugin.version>
+    <spotless.version>2.43.0</spotless.version>
+  </properties>
+
   <developers>
     <developer>
```

Comment:
Please add yourself to the list of developers if you feel like it.

Reply:
I'm good. If you want to add me, that's fine, but I don't really have a horse in this particular race.